tests/pipeline/test_text-generation.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+import time
+import unittest
+
+import torch
+from transformers import AutoConfig, AutoModelForCausalLM, AutoTokenizer, pipeline
+
+from optimum.intel import inference_mode as ipex_inference_mode
+from parameterized import parameterized
+
+
+MODEL_NAMES = {
+    "bloom": "bigscience/bloom-7b1",
+    "gptj": "EleutherAI/gpt-j-6B",
```

Comment:
For tests, please use smaller models instead :
```suggestion
    "bloom": "hf-internal-testing/tiny-random-bloom",
    "gptj": "hf-internal-testing/tiny-random-gptj",
```